### ISSUE TITLE
Control individual corner radius

### ISSUE THREAD

**Issue by samiresua** [contentful/experience-builder#564]
Are there plans to introduce the ability to control each corner radius (top, right, bottom, left) individually?

**Comment by ryunsong-contentful** [comment]
Yes, we have intentional plans of adding this feature in the relative near future!

**Comment by primeinteger** [comment]
@samiresua This feature is now implemented in Studio Experiences! 🎉
